statistics-reports.md

```diff
@@ -0,0 +1,193 @@
+# Displaying usage statistics
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Statistics for a DSpaceObject
+**GET /api/statistics/usage-reports**
+
+This endpoint provides a paginated list of statistics for a DSpaceObject. 
+
+The DSpaceObject is given through the following parameters:
+- `scope` The UUID of the DSpaceObject
+- `scopeType` The string type of the DSpaceObject 
+
+The usual parameters for paginated lists are supported as well:
+- `page` The page number 
+- `size` The number of statistics in a page
+
+An example JSON response document to `/api/statistics/usage-reports?page=0&size=2&scopeType=site&scope=6d65c6a2-3fe7-44dd-bacb-79271257c35d`:
```

Comment:
This endpoint seems to act like a `/search` (https://github.com/DSpace/Rest7Contract/blob/master/search-rels.md), so I think we might want to restructure this as:
`/api/statistics/usage-reports/search/object?uri=`  

This would be similar to how we implemented a search by object in the authorizations endpoint here: https://github.com/DSpace/Rest7Contract/blob/master/authorizations.md#object

This would also likely mean the root endpoint here (/usage-reports) should return a 405 error code, as we don't allow access to all usage-reports at once.

Reply:
Good point, I've adjusted it, and also moved it underneath the single statistic to keep the order in line with the other rest contracts